scripts/cfengine-build-host-setup.cf

```diff
@@ -0,0 +1,199 @@
+body file control
+{
+  inputs => { "$(sys.libdir)/stdlib.cf" };
```

Comment:
OK, so `$(sys.libdir)` means we expect `/var/cfengine/inputs/lib/stdlib.cf` to exist when this policy runs as root or `~/.cfagent/inputs/lib/stdlib.cf` if unprivileged (which cant do much).

Reply:
yes. I am running this policy as root and have installed masterfiles to `/var/cfengine/inputs`